docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -595,6 +595,48 @@ has no DHCP server at all. Therefore, the VM won't have the search domains infor
 reaching a destination using its FQDN is not possible.
 Tracking issue - https://github.com/kubevirt/kubevirt/issues/7184
 
+### passt
+
+`passt` is a new approach for user-mode network that can be used as a simple replacement for Slirp.
+
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: red
+          passt: {} # connect using passt mode
+  networks:
+  - name: red
+    pod: {}
+```
+
+#### Requirements:
+1. To get better performance the node should be configured with:
+```
+sysctl -w net.core.rmem_max = 33554432
+sysctl -w net.core.wmem_max = 33554432
+```
+2. To run multiple passt VMs with no explicit ports, the node's `fs.file_max` should be increased
+   (at least ~2^18 per passt with no explicit ports VM):
+```
+sysctl -w fs.file_max = 9223372036854775807
```

Comment:
How many VMs this allows for?

Reply:
@AlonaKaplan maybe its just me, Could you clarify what this 2^18 magic number actually is ?

Reply:
2^16 is the max port range. You need to open a socket per port (TCP IPv4, TCP IPv6, UDP IPv4, UDP IPv6). So 2^16 *4.
Added this ` for a VM forwards all IPv4 and IPv6 ports, for TCP and UDP, passt needs to create ~2^18 sockets` to the doc. I don't think the exact calculation is needed here.

Reply:
Thanks for explaining.